core/src/main/scala/kafka/utils/ZkUtils.scala

```diff
@@ -522,7 +522,7 @@ class ZkUtils(val zkClient: ZkClient,
       zkClient.delete(path, expectedVersion)
       true
     } catch {
-      case e: KeeperException.BadVersionException => false
+      case e: ZkBadVersionException => false
```

Comment:
Can you take the chance and fix the scaladoc for this method? It says "otherwise (the current version is not the expected version)", but the parenthesis there don't seem to make sense.